Cargo.toml

```diff
@@ -30,6 +30,12 @@ std = ["alloc", "serde?/std"]
 alloc = ["serde?/alloc"]
 derive = ["bincode_derive"]
 
+## !! Not for public use !!
+# features to check again in CI
+ci = ["std", "alloc", "derive", "serde"]
+# experimental strict OOM checks, requires nightly features
+unstable-strict-oom-checks = ["alloc"]

```

Comment:
This needs an entry in the documentation. Make sure to document that using this feature opts you out of all stability guarantees.

Reply:
Done